LayoutTests/TestExpectations

```diff
@@ -7458,15 +7454,34 @@ imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-requ
 # These cross-window tests won't work on ports that don't run on the web-platform.test domains (not glib).
 imported/w3c/web-platform-tests/navigation-api/navigate-event/cross-window/ [ Skip ]
 
+# webkit.org/b/282609
+imported/w3c/web-platform-tests/navigation-api/navigation-history-entry/entries-after-blob-navigation.html [ Pass Timeout ]
+
+# webkit.org/b/282627
+imported/w3c/web-platform-tests/navigation-api/state/same-document-away-and-back-location-api.html [ Pass Timeout ]
+imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/same-document-away-and-back-location-api.html [ Pass Timeout ]
+
+# Unknown cross document traveral issues
+imported/w3c/web-platform-tests/navigation-api/navigation-methods/back-forward-multiple-frames.html [ Pass Timeout ]
+imported/w3c/web-platform-tests/navigation-api/navigation-methods/traverseTo-detach-between-navigate-and-navigatesuccess.html [ Timeout ]
+imported/w3c/web-platform-tests/navigation-api/per-entry-events/dispose-for-full-session-history.tentative.html [ Pass Timeout ]
+imported/w3c/web-platform-tests/navigation-api/per-entry-events/dispose-for-navigation-in-child.html [ Pass Timeout ]
+
+# This one is fixed in https://github.com/WebKit/WebKit/pull/35286
```

Comment:
If this is comment true, then why are we expecting an intermittent timeout for this bug?

Reply:
Ah that PR was closed, not merged. I'll point this to a bug report instead.